src/tupelo.spec.ts

```diff
@@ -120,90 +78,60 @@ describe('Tupelo', () => {
         resolve(true)
       })
     return p
-  }).timeout(10000)
-
-  it('gets envelope bits', async () => {
-    const key = await EcdsaKey.generate()
+  })
 
-    const topic = "alongertopicworks"
+  it('gets token payload', async () => {
```

Comment:
Every time I read a test like this my brain goes, `it('gets the hose again', ...)` 😶

Reply:
https://www.youtube.com/watch?v=20z4edLGxHE